vars/buildSnapshotFabric8UI.groovy

```diff
@@ -29,7 +29,7 @@ def call(body) {
         export FABRIC8_SSO_API_URL="https://sso.prod-preview.openshift.io/"
         export FABRIC8_AUTH_API_URL="https://auth.prod-preview.openshift.io/api/"
         
-        export OPENSHIFT_CONSOLE_URL="https://console.starter-us-east-2.openshift.com/console/"
```

Comment:
how do you know this should be changed and to this value?

Reply:
That value is incorrect - changing to the correct free-stg value will at least help us identify where it's broken in the deploy so it's consistent.